src/js/renderers/flash.js

```diff
@@ -237,6 +237,7 @@ const FlashMediaElementRenderer = {
 			}
 			;
 		methods.push('stop');
+		methods.push('canplay');
```

Comment:
Once you perform this change, and you have the issue solved with progress bar, I'll review it entirely